src/Esfa.Vacancy.UnitTests/CreateApprenticeship/Application/GivenACreateApprenticeshipRequestValidator/AndWageTypeCustom/WhenValidatingWageTypeReason.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Esfa.Vacancy.Application.Commands.CreateApprenticeship;
+using Esfa.Vacancy.Application.Commands.CreateApprenticeship.Validators;
+using Esfa.Vacancy.Domain.Validation;
+using FluentAssertions;
+using NUnit.Framework;
+using Ploeh.AutoFixture;
+
+namespace Esfa.Vacancy.UnitTests.CreateApprenticeship.Application.GivenACreateApprenticeshipRequestValidator.AndWageTypeCustom
+{
+    [TestFixture]
+    public class WhenValidatingWageTypeReason : CreateApprenticeshipRequestValidatorBase
+    {
+        [Test]
+        public async Task AndHasValueThenIsInvalid()
+        {
+            var fixture = new Fixture();
+            var request = new CreateApprenticeshipRequest
+            {
+                WageType = WageType.Custom,
+                WageTypeReason = fixture.Create<string>()
+            };
+
+            var context = GetValidationContextForProperty(request, req => req.WageTypeReason);
+
+            var validator = fixture.Create<CreateApprenticeshipRequestValidator>();
+
+            var result = await validator.ValidateAsync(context);
+
+            result.IsValid.Should().Be(false);
+            result.Errors.First().ErrorCode
+                .Should().Be(ErrorCodes.CreateApprenticeship.WageTypeReason);
+            result.Errors.First().ErrorMessage
+                .Should().Be("'Wage Type Reason' must be empty.");
```

Comment:
`'Wage Type Reason' can't be supplied for the selected WageType`?

Reply:
its the auto gen'd message